packages/redux/src/products/reducer/recentlyViewedProducts.ts

```diff
@@ -54,7 +54,10 @@ const result = (
       return {
         remote: action.payload,
         pagination: omit(action.payload, 'entries') as RecentlyViewedProducts,
-        computed: uniqBy([...action.payload.entries, ...computed], 'productId'),
+        computed: uniqBy(
+          sortBy([...action.payload.entries, ...computed], 'lastVisitDate'),
```

Comment:
Although this might work, since `lastVisitDate` is a string it is more secure to work with the underlying Date object. You can do it like this: 

```suggestion
          sortBy([...action.payload.entries, ...computed], (entry) => new Date(entry.lastVisitDate)),
```

Reply:
Fixed 👍🏼